cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,24 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
+    cy.visit('https://conduit.mate.academy/user/login');
 
+    cy.get('h1')
+      .should('contain.text', 'Sign in');
+
+    // email entering
+    cy.get('#email')
+      .type('bybeg@mailinator.com');
+
+    // password entering
+    cy.get('#password')
+      .type('Pa$$w0rd!');
```

Comment:
Same as above, avoid hardcoding test data directly in the test case. Consider using a data file or a fixture to store your test data.